Makefile

```diff
@@ -71,7 +71,7 @@ help:
 	@echo "test - run python tests with the default Python"
 	@echo "test-deps - install the test dependencies"
 	@echo "test-js-karma - run newer js tests in terminal; primarily for CI build"
-	@echo "test-mocha-karma - run older js tests that have transitioned to karma in the terminal"
+	@echo "test-js-old - run older js tests that have transitioned to karma in the terminal"
```

Comment:
I'd like it if both targets started with "test-js-" - the reason I changed this is that I can never remember which testing framework is used for which set of tests, so I kept having to run `make help` every time. I think "old" is still fair because we want to move tests out of there, even if we have to maintain them short term.